pkg/cmd/grafana-cli/services/services.go

```diff
@@ -62,43 +66,25 @@ func makeHttpClient(skipTLSVerify bool, timeout time.Duration) http.Client {
 	}
 }
 
-func ReadPlugin(pluginDir, pluginName string) (models.InstalledPlugin, error) {
-	distPluginDataPath := filepath.Join(pluginDir, pluginName, "dist", "plugin.json")
+func ReadPlugin(pluginDir, pluginID string) (plugins.FoundPlugin, error) {
+	pluginPath := filepath.Join(pluginDir, pluginID)
```

Comment:
I came across some older plugins which still use `ID/dist/plugin.json` rather than `ID/plugin.json`. Will this change break with such plugins? Do we maybe want to return a separate error in that case to tell the user it's "unsupported"?

Reply:
Ah yes will double check this - will just add it back if not caught and we can kill it another day!

Reply:
We're covered by this case thanks to the heavy lifting of the `LocalFinder` called within `GetLocalPlugins` 🕺

Reply:
Ah, yes that makes sense! :raised_hands: